plugins/hypervisors/xenserver/src/main/java/com/cloud/hypervisor/xenserver/resource/wrapper/xenbase/CitrixMigrateCommandWrapper.java

```diff
@@ -81,6 +81,13 @@ public Answer execute(final MigrateCommand command, final CitrixResourceBase cit
                 }
                 citrixResourceBase.migrateVM(conn, dsthost, vm, vmName);
                 vm.setAffinity(conn, dsthost);
+
+                final String result = citrixResourceBase.callHostPlugin(conn, "vmops", "destroy_network_rules_for_vm", "vmName", command.getVmName());
```

Comment:
So, the cleanup was never implemented...?

Reply:
Yes, it looks like @rafaelweingartner.

I am not aware if it was or not removing the network rules before 4.11/4.12, but I don't see how it would be removed as the mgmt was not calling `destroy_network_rules_for_vm` from the vmops.

Reply:
Thanks for the explanation